packages/core/src/common/logger-watcher.ts

```diff
@@ -26,4 +26,8 @@ export class LoggerWatcher {
     get onLogLevelChanged(): Event<ILogLevelChangedEvent> {
         return this.onLogLevelChangedEmitter.event;
     }
+
+    logLevelChanged(event: ILogLevelChangedEvent) {
```

Comment:
Can you elaborate why is it needed? Events should be distributed via `LoggerClient`. There is no need to have several ways for the same thing. If some code is not properly registering the client. It should be fixed.

(off-topic) it would be nice communicate what method does, like `fireLogLevelChanged`

Reply:
> Can you elaborate why is it needed? Events should be distributed via LoggerClient. There is no need to have several ways for the same thing. If some code is not properly registering the client. It should be fixed.

The LoggerClient object is a proxy that forwards the event to the frontend, which fires the event in the `LoggerWatcher` in the frontend, which updates the `Logger` objects in the frontend.  However, we also have Logger objects in the backend that we need to notify of the log level change.  The easiest way I found to do it was to inject the backend's `LoggerWatcher` in `BunyanLoggerServer`.  However, I needed a public interface to fire the event in `LoggerWatcher`, hence this method.  I'm open to changing how this work if you know a better way.

> (off-topic) it would be nice communicate what method does, like fireLogLevelChanged

I'll change it.